JSDemos/Demos/Gauges/Overview/jQuery/index.js

```diff
@@ -27,10 +28,18 @@ $(() => {
       minorTickInterval: 10,
     },
     valueIndicator: {
-      type: 'twoColorNeedle',
-      color: 'none',
-      secondFraction: 0.24,
-      secondColor: '#f05b41',
+      indentFromCenter: 55,
+      color,
+      spindleSize: 0,
+      spindleGapSize: 0,
+    },
+    centerTemplate: (gauge, container) => {
+      const content = $(`<svg>
```

Comment:
we can not use this approach (creation elements by parsing string). see [this PR](https://github.com/DevExpress/devextreme-demos/commit/9f43bcb3cf73f90463a163b8b9304c521b9e0d08#diff-f5907593ee841b057984b53b6d0a9d69c8c20a241e8c751ec4da3a3e4fa258bdR20). 
this approach potential unsafe and we do not want to show this approach in demos